### PR TITLE
Moved the metadata into setup.cfg

### PR DESCRIPTION
**Pull request by KOLANICH**

### CODE REVIEW

**Comment by wbolster** [comment]
this looks cleaner for sure. can you point me to some resources so that i can better understand the benefits of this approach?

**Comment by KOLANICH** [comment]
@wbolster setuptools docs [describe the format](https://setuptools.readthedocs.io/en/latest/setuptools.html#configuring-setup-using-setup-cfg-files), it's a setuptools feature added in december 2016. The benefits are obvious - no need to execute code in order to get metadata.

**Comment by KOLANICH** [comment]
@wbolster, are you going to merge this?

File: setup.cfg [hunk 1]
```diff
@@ -1,8 +1,32 @@
...
-
 [metadata]
+name = jsonlines
+version = 1.2.0
+author = Wouter Bolsterlee
+author_email = wouter@bolsterl.ee
+license = BSD
 license_file = LICENSE.rst
+description = Library with helpers for the jsonlines file format
+long_description = file: README.rst
```
Comment (wbolster) [inline setup.cfg]: is the space after `file:` allowed?

**Comment by wbolster** [comment]
thanks, i was worried about compatibility but december 2016 seems old enough 😃

File: setup.cfg (hunk 1 above)
Comment (KOLANICH) [reply setup.cfg]: it is required.

**Comment by wbolster** [comment]
thanks, looks good!

**Pull request merged**